atlp-rwanda/vikings-ec-fe
# Fix sales, orders and payments

### What does this PR do?
This PR implements the fixes for the search bar, payment, orders and sales

### Description of Task to be completed?

- [x] Don't show search bar for sellers
- [x] Fix the payment success page
- [x] Sort the orders
- [x] Add pagination on sales

### How should this be manually tested?

1. Clone [repo](https://github.com/atlp-rwanda/vikings-ec-fe.git)
2. Change into the directory **vikings-ec-fe**
3. Checkout to this branch:` fix/payment-orders`
4. Login with user:`admin@gmail.com` and password:`Password@123`
5. Visit the sales page and check the pagination.
6. Visit the orders page and you will see that the orders are sorted with the newest at the top
7. Login as a seller and you won't see the searchbar
